### PR TITLE
test: add test to ensure governance token is minted every block

### PR DESCRIPTION
**Pull request by Pangssu**
Added tests related to mint a `GovernanceToken` via `L1Block` and `MintManager` contracts.

- Test if the`Transfer` event of the `GovernanceToken` is present within the L1 Attribute Deposited Transaction.
- Test if token minting has started on an activated block recorded in the `MintManager` contract.
- Test if the minting of the initial issuance is carried out and distributed correctly.
- Test if the token issuance amount decreases over time and eventually depletes.
- Test if the `GovernanceToken` can be bridged between L1 and L2.

## Summary by CodeRabbit

- **Tests**
	- Introduced comprehensive tests for token minting under various conditions and states.
	- Added tests for L1StandardBridge bridge GovernanceToken functionality, covering token transfers, balances, and cross-network transactions.
	- Enhanced system tests with configurations for minting governance tokens and verifying functionality within contracts.

### CODE REVIEW

**Comment by Pangssu** [comment]
@coderabbitai review

File: op-e2e/bridge_test.go [hunk 1]
```diff
@@ -119,3 +125,129 @@ func TestERC20BridgeDeposits(t *testing.T) {
...
+	_, err = geth.WaitForBlock(big.NewInt(10), l2Client, 20*time.Second)
+	require.NoError(t, err)
+	aliceL2Opts, err := bind.NewKeyedTransactorWithChainID(cfg.Secrets.Alice, cfg.L2ChainIDBig())
+	require.Nil(t, err)
+	bridgeAmount, err := l2Token.BalanceOf(&bind.CallOpts{}, aliceL2Opts.From)
+	require.NoError(t, err)
+	require.NotZero(t, bridgeAmount.Uint64())
+
+	// Send Alice's tokens to Bob
+	require.Nil(t, err)
```
Comment (0xHansLee) [inline op-e2e/bridge_test.go:190]:
```suggestion
```

File: op-e2e/bridge_test.go [hunk 2]
```diff
@@ -119,3 +125,129 @@ func TestERC20BridgeDeposits(t *testing.T) {
...
+	require.NoError(t, err)
+	version := eth.OutputVersionV0
+	proveReceipt, finalizeReceipt := ProveAndFinalizeWithdrawal(t, version, cfg, l1Client, sys.EthInstances["verifier"], cfg.Secrets.Bob, receipt)
+	require.Equal(t, types.ReceiptStatusSuccessful, proveReceipt.Status)
+	require.Equal(t, types.ReceiptStatusSuccessful, finalizeReceipt.Status)
+
+	// Withdrawal complete, Should have balance on L1
+	bobL1Balance, err = l1Token.BalanceOf(&bind.CallOpts{}, l1Opts.From)
+	require.NoError(t, err)
+	require.NotZero(t, bobL1Balance.Uint64())
```
Comment (0xHansLee) [inline op-e2e/bridge_test.go:220]: Please add check the L2 balance of bob is zero and total supply on L2 is decreased. I'm not sure the latter can be done in this e2e.

File: op-e2e/bridge_test.go [hunk 3]
```diff
@@ -119,3 +125,129 @@ func TestERC20BridgeDeposits(t *testing.T) {
...
+
+	depositTx, err := derive.UnmarshalDepositLogEvent(&depositEvent.Raw)
+	require.NoError(t, err)
+	_, err = wait.ForReceiptOK(context.Background(), l2Client, types.NewTx(depositTx).Hash())
+	require.NoError(t, err)
+
+	// Should have balance on L2
+	bobL2Balance, err = l2Token.BalanceOf(&bind.CallOpts{}, l2Opts.From)
+	require.NoError(t, err)
+	require.Equal(t, bobL1Balance, bobL2Balance)
```
Comment (0xHansLee) [inline op-e2e/bridge_test.go:252]: And please check the balance of Bob on L1 is zero.
Further more, let's check the total supply on L1 is decreased by the amount of deposit.

**Review by 0xHansLee — APPROVED** [review]
LGTM

**Review by sm-stack — APPROVED** [review]
LGTM

**Review by kangsorang — APPROVED** [review]

**Pull request merged**